scripts/e2e-tests.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 set -e
 
-TEST_ARG="${1:-*}"
-
-MYDIR=$(realpath "$(dirname "$0")")
-cd "$MYDIR"
-cd ..
+# Navigate to project root
+cd "$(dirname "$(realpath "$0")")/.."
```

Comment:
Consider that reduction of code lines, doesn't reduce code complexity.
After all we could feed all code as a single line to compiler and he will be fine.

p.s. If you think that new changes are better lets keep them

Reply:
To me it's the same, as the dirname+realpath combo is (un)clear enough.

Though now that I think about it, wouldn't it be enough to just do `cd ..`? :thinking:

Reply:
I see it was necessary before:D or just a good tone of writing repo wide scripts. Step to root directory first

Reply:
Lets use then this construct in other scripts
```
# Move to project root
cd "$(dirname "$(realpath "$0")")/.."
```

Reply:
My bad, it is actually better to ensure we're in the project root :sweat_smile: 

I don't mind reverting this to the way it was before, to me it's the same. Will do so.